README.md

```diff
@@ -17,24 +17,76 @@ on [Qcodes](https://github.com/qdev-dk/Qcodes) (basic framework such as instrume
 
 ## Installation
 
-QTT is compatible with Python 3.5+. QTT can be installed as a pip package:
+QTT is compatible with Python 3.5+. QTT can be installed as a pip package to be used in a (virtual) Python environment.
+
+### Setting up a virtual environment
+To create a clean virtual Python environment for your qtt development do:
+```
+$ mkdir qtt
+$ cd qtt
+$ python -m venv env
+```
+Now activate the virtual environment. On Linux do:
+```
+$ . ./env/Scripts/activate
```

Comment:
I also tried it on ubuntu with a virtual machine. For it to work I needed to do the following:
```
sudo apt install git python3 python3-venv python3-dev gcc
python -m venv env
git clone git@github.com:qutech-delft.qtt.git
pip install wheel
pip install -e .
```

Should we add this as well?

Reply:
I added a sentence that software as python and git should be installed already (with a link to download pages). I added the pip wheel install to the description